deployments/helm-chart/README.md

```diff
@@ -146,111 +146,112 @@ See [running multiple ingress controllers](https://docs.nginx.com/nginx-ingress-
-`controller.service.customPorts` | A list of custom ports to expose through the Ingress Controller service. Follows the conventional Kubernetes yaml syntax for service ports. | []
-`controller.service.httpPort.enable` | Enables the HTTP port for the Ingress Controller service. | true
-`controller.service.httpPort.port` | The HTTP port of the Ingress Controller service. | 80
-`controller.service.httpPort.nodePort` | The custom NodePort for the HTTP port. Requires `controller.service.type` set to `NodePort`. | ""
-`controller.service.httpPort.targetPort` | The target port of the HTTP port of the Ingress Controller service. | 80
-`controller.service.httpsPort.enable` | Enables the HTTPS port for the Ingress Controller service. | true
-`controller.service.httpsPort.port` | The HTTPS port of the Ingress Controller service. | 443
-`controller.service.httpsPort.nodePort` | The custom NodePort for the HTTPS port. Requires `controller.service.type` set to `NodePort`.  | ""
-`controller.service.httpsPort.targetPort` | The target port of the HTTPS port of the Ingress Controller service. | 443
-`controller.serviceAccount.name` | The name of the service account of the Ingress Controller pods. Used for RBAC. | Autogenerated
-`controller.serviceAccount.imagePullSecretName` | The name of the secret containing docker registry credentials. Secret must exist in the same namespace as the helm release. | ""
-`controller.serviceMonitor.name` | The name of the serviceMonitor. | Autogenerated
-`controller.serviceMonitor.create` | Create a ServiceMonitor custom resource. | false
-`controller.serviceMonitor.labels` | Kubernetes object labels to attach to the serviceMonitor object. | ""
-`controller.serviceMonitor.selectorMatchLabels` | A set of labels to allow the selection of endpoints for the ServiceMonitor. | ""
-`controller.serviceMonitor.endpoints` | A list of endpoints allowed as part of this ServiceMonitor. | ""
+Parameter | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Default
+--- |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ---
```

Comment:
This line appears to have changed the entire formatting for the README.md file. Can correct the formatting for this file.

Reply:
Also, I've fixed that line

Reply:
@0m1xa seems like there's still some weird formatting to the whole file, instead of just a change with your addition. I feel like it might be your IDE.

Reply:
@lucacome you're right. It was my IDE's formatting. Sorry for that.
Now it is fixed.